pkg/virtctl/root.go

```diff
@@ -36,6 +36,29 @@ func NewVirtctlCommand() *cobra.Command {
 			fmt.Fprint(cmd.OutOrStderr(), cmd.UsageString())
 		},
 	}
+	completionCmd := &cobra.Command{
```

Comment:
I think you can copy my exact descriptions from #916. It is itself a copy from `kubectl complete`.